Software/Latex Docs/Software Steps/Software Steps.tex

```diff
@@ -203,75 +199,39 @@ \subsection{Setting up serial communication}
 \begin{itemize}
-	\item \href{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}. 
+	\item \href{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}{https://spellfoundry.com/2016/05/29/configuring-gpio-serial-port-raspbian-jessie-including-pi-3/}.
 \end{itemize}
 \noindent Once you've completed the above steps, reboot the Pi.
 \begin{itemize}
 	\item[] \colorbox{lightgray}{sudo reboot}
 \end{itemize}
 
-\subsection{Downloading the Rover Code}
+\subsection{Building the Rover Code to work with your ROS installation}
 On the Raspberry Pi, open up a terminal \textbf{(ctl + alt + t)} and then type the following commands\footnote{In this document, terminal commands will be \colorbox{lightgray}{highlighted}.}:
 
-\begin{itemize}
-	\item[] Move to your home directory: \colorbox{lightgray}{cd /home/pi}
-	\item[] Clone the rover code repository: \colorbox{lightgray}{git clone https://github.com/nasa-jpl/osr-rover-code osr}
-	\item[] Move into the directory you just cloned: \colorbox{lightgray}{cd osr}
-	\item[] Pull to make sure you have the latest code: \colorbox{lightgray}{git pull}
-\end{itemize}
-
-\subsection{Building the Rover Code to work with your ROS installation}
-Now, you must build the rover code packages so that ROS can run them on your Raspberry Pi.  To do this, run the following commands in a terminal:
+Create a catkin workspace which will contain all ROS compilation and source code files:
 
 \begin{itemize}
 	\item[] Create a catkin workspace directory and move into it:
-	\item[] \colorbox{lightgray}{mkdir -p /home/pi/osr\_ws/src \&\& cd /home/pi/osr\_ws}
+    \item[] \colorbox{lightgray}{mkdir -p \url{~/osr_ws/src} \&\& cd \url{~/osr\_ws}}
 	\item[] Build a basic, empty catkin project: \colorbox{lightgray}{catkin\_make}
-	\item[] Move to the directory containing the main OSR logic: \colorbox{lightgray}{cd /home/pi/osr/ROS}
-	\item[] Copy the OSR source files to the catkin workspace:
-	\item[] \colorbox{lightgray}{mv led\_screen/ osr/ osr\_bringup/ osr\_msgs/ /home/pi/osr\_ws/src}
-	\item[] Make all of the source files executable:  \colorbox{lightgray}{cd /home/pi/osr\_ws/src/osr/scripts \&\& chmod +x *.py}
-	\item[] Move back to your catkin workspace: \colorbox{lightgray}{cd /home/pi/osr\_ws}
-	\item[] Build the sensor\_msgs ROS package:
-	\item[] \colorbox{lightgray}{\parbox{0.9\textwidth}{rosinstall\_generator sensor\_msgs --rosdistro kinetic --deps --wet-only --tar \textgreater kinetic-sensor\_msgs-wet.rosinstall}}
-	\item[] \colorbox{lightgray}{wstool init src kinetic-sensor\_msgs-wet.rosinstall}
-	\item[] Run a "catkin make" of the OSR workspace (this will take some time): \colorbox{lightgray}{catkin\_make}
+	\item[] If this doesn't report any errors, check if there are two new directories \url{~/osr_ws/build} and  \url{~/osr_ws/devel}
```

Comment:
The .pdf doesn't render these as commands, i.e. we should probably list these as `ls ~/osr_ws/build` and `ls ~/osr_ws/devel` instead of how it reads now:

![image](https://user-images.githubusercontent.com/2334865/83209476-53c09c00-a10d-11ea-9836-6738e4bc9852.png)

Remember that this may _literally_ be someone's first time in a terminal.  They may not yet know how to "check if there are two new directories" on the pi without a GUI.